src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1718,6 +1751,62 @@ private static async Task ExecuteGraphQLIntrospectionQueries(TestServer server,
             }
         }
 
+        private static JsonContent GetJsonContentForCosmosConfigRequest(string endpoint, string config = null, bool useAccessToken = false)
+        {
+            if (CONFIGURATION_ENDPOINT == endpoint)
+            {
+                ConfigurationPostParameters configParams = GetCosmosConfigurationParameters();
+                if (config != null)
+                {
+                    configParams = configParams with { Configuration = config };
+                }
+
+                if (useAccessToken)
+                {
+                    configParams = configParams with
+                    {
+                        ConnectionString = "AccountEndpoint=https://localhost:8081/;",
+                        AccessToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiZXhwIjoxMjMzNDQ1Nn0.1cdRZfqwndt67f-sHKgOfEgTfO9xDyGFl6_d-RRyf4U"
```

Comment:
is this the well known emulator access token? Does Static Analysis have any issues with checking this in? its password like hence wondered

Reply:
No this was a random token with no information in it: 
> {
  "alg": "HS256",
  "typ": "JWT"
}.{
  "sub": "1234567890",
  "name": "John Doe",
  "exp": 12334456
}.[Signature]

But I've replaced it with generating one during the test since it could trip static analysis tools. Thanks for pointing this out.